tracer/test/test-applications/security/aspnet/Samples.Security.AspNetMvc5/Controllers/IastController.cs

```diff
@@ -354,6 +354,7 @@ public ActionResult GetFileContentEdgeConditions(string file, bool endRequest =
 
             if (endRequest)
             {
+                Response.Flush();
```

Comment:
did you mean to change this?

Reply:
Yes, this is a method that is used for manual testing and calling the method flush is a way to set HeadersWritten to true. This is something that a customer can potentially do and could provoke the exception.